### PR TITLE
Glovo_db commit

### PR DESCRIPTION
**Pull request by ZarubeyAleksandr**

### CODE REVIEW

**Comment by Kateryna2882** [comment]
Good job!

**Review by Kateryna2882 — APPROVED** [review]

File: src/main/java/org/glovo/glovo_db/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        updatedOrder.setId(id);
+        orderService.updateOrder(updatedOrder);
+    }
+
+    @DeleteMapping("/{id}")
+    public void deleteOrder(@PathVariable Long id) {
+        orderService.deleteOrder(id);
+    }
+
+    @PostMapping("/{id}/addProduct")
```
Comment (JzomitiZumatti) [inline src/main/java/org/glovo/glovo_db/controller/OrderController.java]: According to task, you need to use @Patch

**Review by JzomitiZumatti — APPROVED** [review]

File: src/main/java/org/glovo/glovo_db/controller/OrderController.java (hunk 1 above)
Comment (ZarubeyAleksandr) [reply src/main/java/org/glovo/glovo_db/controller/OrderController.java]: yes, thanks. check

File: src/main/java/org/glovo/glovo_db/controller/OrderController.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        return orderService.getOrderById(id);
+    }
+
+    @PostMapping
+    public Order createOrder(@RequestBody Order order) {
+        return orderService.createOrder(order);
+    }
+
+    @PutMapping("/{id}")
+    public void updateOrder(@PathVariable Long id, @RequestBody Order updatedOrder) {
```
Comment (mkharkhuta) [inline src/main/java/org/glovo/glovo_db/controller/OrderController.java]: It's better to use separate class OrderUpdateRequest with only the fields to update

**Review by mkharkhuta — COMMENTED** [review]
Good! Next time, please ass pom.xml

File: src/main/java/org/glovo/glovo_db/controller/OrderController.java (hunk 2 above)
Comment (ZarubeyAleksandr) [reply src/main/java/org/glovo/glovo_db/controller/OrderController.java]: check

**Pull request merged**